### PR TITLE
add wicket v1.3.1 with npm auto-update config

### PR DESCRIPTION
**Pull request by kennynaoh**
PR for #6746
 @x09326  please help me review this pull request, thank you.
Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/arthur-e/Wicket
 * Official website (optional, not the repository): https://arthur-e.github.io/Wicket
 * NPM package url (optional): https://www.npmjs.com/package/wicket
 * GitHub / Bitbucket popularity (required):
   - Count of stars: 181
   - Count of watchers: 22
   - Count of forks: 76
 * NPM download stats (optional):
   - Downloads in the last day: 41
   - Downloads in the last week: 70
   - Downloads in the last month: 197

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [ ] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [ ] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [ ] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

cc #6746, cc @arthur-e

### CODE REVIEW

**Comment by x09326** [comment]
@kennynaoh 
Please add ```"author"``` and ```"homepage"``` fields in the **package.json**.
And I think other files should be minified.

**Comment by kennynaoh** [comment]
@x09326 
I have updated it.
Please help me review this PR again.
Thanks.

**Comment by arthur-e** [comment]
Hi, the "Official website (optional, not the repository)" needs to be changed. It is:

https://arthur-e.github.io/Wicket/

**Comment by PeterDaveHello** [comment]
@arthur-e Thanks.

**Comment by kennynaoh** [comment]
@x09326 
I have updated it.
Please help me review it again.
Thanks.

File: ajax/libs/wicket/package.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    "map",
+    "mapping",
+    "geometry",
+    "leaflet"
+  ],
+  "author": "arthur-e",
+  "homepage": "https://arthur-e.github.io/Wicket",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/arthur-e/Wicket"
```
Comment (x09326) [inline ajax/libs/wicket/package.json]: ```"url"``` should be ```"https://github.com/arthur-e/Wicket.git"```

Comment (arthur-e) [inline ajax/libs/wicket/package.json]: I have just fixed this in `master`.

**Comment by kennynaoh** [comment]
Thanks all.
@x09326 
I have updated it.
Please help me review it again.
Thanks.

File: ajax/libs/wicket/package.json (hunk 1 above)
Comment (PeterDaveHello) [inline ajax/libs/wicket/package.json]: @x09326 https://github.com/arthur-e/Wicket is fine, no need to modify it.

**Comment by x09326** [comment]
@PeterDaveHello 
I think this PR is okay, please help me review it, thanks.

**Comment by PeterDaveHello** [comment]
You didn't mention the related issue in the commit message.

**Comment by kennynaoh** [comment]
@x09326 
I have updated my commit message.
Please help me review it again.
Thanks

**Comment by x09326** [comment]
@Peterdavehello 
I think this pr is okay.
Please help me review it, thanks.

**Comment by PeterDaveHello** [comment]
You didn't rebase the branch with the latest master branch... @kennynaoh I wonder if you are too tired, keep missing some small things.

**Comment by kennynaoh** [comment]
Deeply sorry about that.
And I have rebased already.
By the way, I want to ask something.
I am not sure about the time of the rebase.
Because I saw this PR was 36 commits behind cdnjs:master.
I want to ask what time I need to rebase when the commits number more than one number.
Thanks.

**Comment by PeterDaveHello** [comment]
@kennynaoh As I said before, you should do it everytime you update your commit in a PR.

**Comment by kennynaoh** [comment]
OK, Thanks.
@x09326 
Please help me review this PR again.
Thanks.

**Pull request merged**